src/main/java/Events.java

```diff
@@ -0,0 +1,22 @@
+public class Events extends Task {
+    protected String by;
+    Events(String name, String by) {
+        super(name);
+        this.by = by;
+    }
+
+    /**
+     * Return the saved Format of Events task
+     * @return
```

Comment:
For @return tag in javadoc, should include description on what the method is returning